lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -616,6 +616,24 @@ void remapSublayerRecursive(
     }
 }
 
+void reproduceSharedStageState(
+    const UsdStageRefPtr& finalUsdStage,
+    const SdfLayerRefPtr& inRootLayer,
+    const SdfLayerRefPtr& unsharedStageRootLayer)
+{
+    // Transfer the FPS (frames-per-second) of the original root layer to the new unshared
+    // root layer, so that the animation timeline does not change. We copy both the metadata
+    // on the layer and on the stage object itself.
+    if (SdfDataRefPtr metadata = inRootLayer->GetMetadata()) {
+        VtValue fpsValue;
+        if (metadata->Has(SdfPath("/"), TfToken("framesPerSecond"), &fpsValue)) {
+            unsharedStageRootLayer->GetMetadata()->Set(
```

Comment:
A few lines above you check that GetMetadata() is not null, and here you don't. Shouldn't we be more consistent?

Reply:
Good catch! Although I will actually check the USD docs to see if it is possible for the metadata to be null. So I'll either add a check or remove the other one.